.github/workflows/version-auto-bump.yml

```diff
@@ -0,0 +1,121 @@
+---
+name: Version Auto Bump
+
+on:
+  release:
+    types: [published]
+
+
+jobs:
+
+  setup:
+    name: "Setup"
+    runs-on: ubuntu-20.04
+    outputs:
+      version_number: ${{ steps.version.outputs.new-version }}
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+
+      - name: Get version to bump
+        id: version
+        run: |
+          CURR_MAJOR=$(echo ${{ github.event.release.tag }} | sed -r 's/v([0-9]{4}\.[0-9]\.)([0-9])/\1/'
+          CURR_VER=$(echo ${{ github.event.release.tag }} | sed -r 's/v([0-9]{4}\.[0-9]\.)([0-9])/\2/'
+          echo $CURR_VER
+
+          ((CURR_VER++))
+          NEW_VER=$CURR_MAJOR$CURR_VER
+
+          echo "::set-output name=new-version::$NEW_VER"
+
+  bump_version:
```

Comment:
Is it possible for to trigger the version bump workflow with the needed inputs? Similar to how we do it in `build-browser.yml`?
https://github.com/bitwarden/clients/blob/12bca0a597dada622e23a37830da0c266b0c8048/.github/workflows/build-browser.yml#L348

Reply:
That's a great idea to trigger the version bump workflow from the new auto bump workflow.